### PR TITLE
DEP-8 Specify that generated code will be Black formatted.

### PR DESCRIPTION
**Pull request by adamchainz**
I suggest this small ("small"?) extra that will make user experience more consistent. It should be possible to implement it by calling black on generated code before saving it to a file.

### CODE REVIEW

File: accepted/0008-black.rst [hunk 1]
```diff
@@ -73,6 +73,10 @@ do, for example channels_ and asgiref_.
 .. _channels: https://github.com/django/channels
 .. _asgiref: https://github.com/django/asgiref
 
+ALl code Django generates will also be Black-formatted
```
Comment (charettes) [inline accepted/0008-black.rst:76]:
```suggestion
All code Django generates will also be Black-formatted
```

**Comment by MarkusH** [comment]
Since the DEP has already been accepted, do we need to do another round of approvals?

**Comment by MarkusH** [comment]
> In order to be consistent should we also use `isort` on generated files as well if it's installed?

I don't think we should mix in the usage of `isort` in a DEP about `black`. Also, isort has significantly more config options than black, that can vary significantly.

**Comment by adamchainz** [comment]
I agree we should avoid isort here at least.

I'll note there is "black-flavoured isort" from a suggested config file in the [black docs](https://black.readthedocs.io/en/stable/the_black_code_style.html), which reduces the options to configure.

**Comment by andrewgodwin** [comment]
I think this is sensible, especially for migration files, as they are the ones most often made during a project, and I think this falls under clarifying the original DEP, not requiring a new one.

Let's not tackle isort for now, though.

**Comment by adamchainz** [comment]
Thanks all

**Comment by carltongibson** [comment]
> Let's not tackle isort for now, though.

A wee bit late to the party but, we already use isort — I don't see that updating the options to ensure that it doesn't fight with black is either out of scope for this DEP, or in need of further approval.

**Comment by adamchainz** [comment]
I thought we were only discussing `isort`-ing files generated by Django. I think it's implied that by adopting Black, we'll have to adopt the compatible `isort` config to stop them fighting.

**Comment by carltongibson** [comment]
Ah, OK. Narrower focus. Thanks. (Still... 🙂)

**Pull request merged**